.palantir/revapi.yml

```diff
@@ -1024,6 +1024,38 @@ acceptedBreaks:
       old: "class org.apache.iceberg.types.Types.NestedField"
       new: "class org.apache.iceberg.types.Types.NestedField"
       justification: "new Constructor added"
+    org.apache.iceberg:iceberg-core:
```

Comment:
Do we still need to add these rules if the break is justified (going to the deprecation process), or am I overestimating the abilities of rev-api here? :D

Reply:
yes we still need to add these here

Reply:
I think in Trino we somehow avoid the need for rev-api exclusions, because we instructed it to watch for `@Deprecated` annotation. However, we would need something more elaborate to take into account the text associated with the `@deprecated` javadoc, which announces the version in which the given element is supposed to disappear. I don't know whether rev-api can already look at the javadoc text, but it definitely would be nice.